docs/SECURITY.md

```diff
@@ -1 +1,19 @@
 # CWA-Server Security
+
+## TLS Configuration
+
+The cwa-server uses encrypted connections for external and internal communication. The main configuration for the external communication channel can be found [here](https://github.com/corona-warn-app/cwa-server/blob/36ac6f7cbcf9b230d6af8ddea96b387b47a3dee9/services/submission/src/main/resources/application.yaml#L76). The current settings will only allow TLS version 1.2. and 1.3. The offered ciphers are chosen in accordance to the recommendations in [BSI-TR-02102-2](https://www.bsi.bund.de/SharedDocs/Downloads/DE/BSI/Publikationen/TechnischeRichtlinien/TR02102/BSI-TR-02102-2.pdf).
+Internal communication is configured for instance [in the feign client](https://github.com/corona-warn-app/cwa-server/blob/master/services/submission/src/main/java/app/coronawarn/server/services/submission/verification/CloudFeignClientProvider.java).
+Database connections are established via TLS and are enabled for mutual authentication, see [here](https://github.com/corona-warn-app/cwa-server/blob/master/services/submission/src/main/resources/application.yaml#L38).
+
+## Security-Related Spring Profiles
+
+The TLS configuration previously described can be changed to enable certain test scenarios. It is important to realize that this degradation of security is only able by setting a Spring profile, while the default behavior will always use the most secure configuration. To prevent misconfiguration in production, multiple controls are implemented. This includes among others configuration-as-code deployments, explicit log messages for unsafe configurations and monitoring of deployed configurations.
+The relevant profiles are:
+
+* `disable-ssl-server` Disables SSL for the submission endpoint.
```

Comment:
Consider switching to a table format to be consistent with our other documentation of profiles (currently in README.md and being moved to overall documentation)